ansible/group_vars/all/piecewise_global_config.yml

```diff
@@ -0,0 +1,278 @@
+
+# Piecewise requires a topojson file for the website to use. This topojson file must
+# be created from the shapefile you defined above. 
+# Save your topojson file into the folder /local_customizations/geofiles
+# and then enter its filename here.
+topojson_file: seattle_census10_blockgroups.topojson
+
+# 'geoKey' is the field name in your topojson file which corresponds to the unique key 
+# in your shapefile, and is typically the same as dbKey, but in CAPITAL LETTERS.
+geoKey: GEOID10
+
+# 'mapscript_start_year' is the first year you are interested in showing in the year dropdown
+# 'mapscript_start_month' lets you define the order of months in the UI.
+mapscript_start_year: 2016
+mapscript_start_month: 1
+
+# 'mapscript_base_layer' defines the type of base layer to be used on the map.
+# Either Mapbox.com or OpenStreetMaps base layers are supported. 
+#
+# Set mapscript_base_layer to either: 'mapbox' OR 'osm'
+mapscript_base_layer: osm
+
+# If you would like to customize the OSM base layer API url, for example if you
+# are hosting custom maps and tiles on your own tile server, update the variable below.
+# The default OSM url is provided.
+mapscript_osm_api_url: "http://{s}.tile.osm.org/{z}/{x}/{y}.png"
+
+# If you are using a Mapbox base layer, set the API URL for your Mapbox tiles here.
+mapscript_mapbox_api_url: "https://api.mapbox.com/styles/v1/newamerica/ciy31olzw00792sp6w98b76i0/tiles/256/{z}/{x}/{y}?access_token=pk.eyJ1IjoibmV3YW1lcmljYSIsImEiOiIyM3ZnYUtrIn0.57fFgg_iM7S1wLH2GQC71g"
+
+# 'mapscript_min_data_points' allows you to set a minimum number of tests which must be 
+# present in any aggregate area defined by your shapefile in order to be displayed on the 
+# aggregate website map. 
+mapscript_min_data_points: 5
+
+# 'mapscript_zoom_level' sets the initial zoom level for the map
+mapscript_zoom_level: 11
+
+# These variables name the map layers on the website. 'mapscript_layer_name' is used as 
+# a layer name in the JavaScript code, and 'mapscript_layer_name_pretty' is used in the 
+# UI if the layer controls are displayed for the user. By default, Piecewise will deploy
+# with one layer, and will not show the layer controls.
+mapscript_layer_name: "census_block_groups"
+mapscript_layer_name_pretty: "Census block groups"
+
+# In the future, support for other types of geo data, such as the geojson format may be
+# included in Piecewise. Currently, the mapscript json type should always be 'topojson'. 
+mapscript_json_type: "topojson"
+
+
+# This dictionary of month names/abbreviations lets you configure how month names appear 
+# on the website map controls. 
+mapscript_month_names: 
+  1: Jan
+  2: Feb
+  3: Mar
+  4: Apr
+  5: May
+  6: Jun
+  7: Jul
+  8: Aug
+  9: Sep
+  10: Oct
+  11: Nov
+  12: Dec
+
+###
+# WEBSITE TEXT CUSTOMIZATIONS
+#
+# The section below allows you to customize various text and other elements of the website
+
+# 'site_name' is used as the HTML page title element
```

Comment:
This might be ok for now, but I'd suggest looking into a more fully fledged i18n tool.  In python [gettext](https://docs.python.org/3/library/gettext.html#gettext) is the tool of choice and has Flask integration in the form of [Flask-Babel](https://pythonhosted.org/Flask-Babel/).